#left stick movements#
float64 l_stick_x_location 0.0
float64 l_stick_y_location 0.0
#right stick movements#
float64 r_stick_x_location 0.0
float64 r_stick_y_location 0.0
#X-Constants of stick movements#
float64 MAX_X=1.0
float64 MIN_X=-1.0
#Y-Constants of righht stick movements#
float64 MAX_Y=1.0
float64 MIN_Y=-1.0